source/features/rgh-feature-descriptions.tsx

```diff
@@ -1,38 +1,42 @@
+import './rgh-feature-descriptions.css';
 import React from 'dom-chef';
-import elementReady from 'element-ready';
 import * as pageDetect from 'github-url-detection';
 
 import features from '../feature-manager';
-import {wrapAll} from '../helpers/dom-utils';
 import {featuresMeta} from '../../readme.md';
 import {getNewFeatureName} from '../options-storage';
 import {isRefinedGitHubRepo} from '../github-helpers';
+import observe from '../helpers/selector-observer';
+
+async function add(infoBanner: HTMLElement): Promise<void> {
+	console.log(infoBanner);
 
-async function init(): Promise<void | false> {
 	const [, currentFeature] = /source\/features\/([^.]+)/.exec(location.pathname) ?? [];
 	// Enable link even on past commits
 	const currentFeatureName = getNewFeatureName(currentFeature);
 	const feature = featuresMeta.find(feature => feature.id === currentFeatureName);
 	if (!feature) {
-		return false;
+		return;
 	}
 
 	const conversationsUrl = new URL('https://github.com/refined-github/refined-github/issues');
 	conversationsUrl.searchParams.set('q', `sort:updated-desc "${feature.id}"`);
 
-	const commit = await elementReady([
-		'.Box-header.Details', // Already loaded
-		'include-fragment.commit-loader', // Deferred loading
-	].join(','));
-
-	const commitInfoBox = commit!.parentElement!;
-
-	commitInfoBox.classList.add('width-fit', 'min-width-0', 'flex-auto', 'mb-lg-0', 'mr-lg-3');
-	commitInfoBox.classList.remove('flex-shrink-0');
-
-	const featureInfoBox = (
-		<div className="Box rgh-feature-description" style={{flex: '0 1 544px'}}>
-			<div className="Box-row d-flex height-full">
+	infoBanner.before(
+		<div className="Box mb-3">
+			<div className="Box-row d-flex gap-3 flex-wrap">
+				<div className="rgh-feature-description">
+					{ /* eslint-disable-next-line react/no-danger */ }
+					<h3 dangerouslySetInnerHTML={{__html: feature.description}}/>
```

Comment:
I also made the text bigger because now we're taking the full width of the page.